daemon/config_linux.go

```diff
@@ -0,0 +1,43 @@
+package daemon
+
+import (
+	"github.com/docker/docker/opts"
+	flag "github.com/docker/docker/pkg/mflag"
+	"github.com/docker/docker/pkg/ulimit"
+)
+
+var (
+	defaultPidFile = "/var/run/docker.pid"
+	defaultGraph   = "/var/lib/docker"
+)
+
+// Config defines the configuration of a docker daemon.
+// These are the configuration settings that you pass
+// to the docker daemon when you launch it with say: `docker -d -e lxc`
+type Config struct {
+	CommonConfig
+
+	// Fields below here are platform specific.
+	EnableSelinuxSupport bool
+	ExecOptions          []string
```

Comment:
how is that different from https://github.com/docker/docker/pull/12770/files#r30067318 ? But I agree with you that ExecOptions and GraphOptions should be in CommonConfig.